front/src/routes.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import App from './App';
+import Home from "./Scenes/Home";
+import Login from "./Scenes/Login";
+import SignUp from "./Scenes/Signup";
+import Services from "./Scenes/Services";
+import ManageWidgets from "./Scenes/ManageWidgets";
+
+
+const routes = (isLoggedIn: boolean) => {
+	const ifLogged = ((ifLogged: any, ifNotLogged:any = <Login/>) => isLoggedIn ? ifLogged : ifNotLogged)
```

Comment:
Maybe use a <Redirect /> to login if the user is not logged in, if it exists in react-router v6

Reply:
To be fixed in a later pr